### PR TITLE
[19.0 Release] Fixing merge issues

### PR DESCRIPTION
**Pull request by ridz1208**
This fixes merge issues created in https://github.com/aces/Loris/pull/3516. The goal here is to not introduce manual commits to a PR which s simply merging the release in the aces/major branch. Instead the fixes are made in this PR based on the Merge PR.

to see the bug fixes alone, follow this link: https://github.com/ridz1208/Loris/compare/2018_02_22_upgrading_major_to_19_take2...ridz1208:2018_02_22_upgrading_major_to_19_take2_FIXES

Bug fixes have been made in accordance to:
 - https://github.com/aces/Loris/pull/2652/files#diff-51cb063f1c365054ed560f9e40a65a84
 - https://github.com/aces/Loris/pull/3510 (specifically https://github.com/kongtiaowang/Loris/commit/ac2311e989d2bdd10b51500baa408af9839f0579)

### CODE REVIEW

**Comment by ridz1208** [comment]
@driusan I'm pretty sure if you merge this one, github will automatically merge both this and its prequel

**Comment by driusan** [comment]
@ridz1208 yes it will, but this one is still failing Travis?

**Comment by ridz1208** [comment]
@driusan not sure why, its passing on 7.1
restarting it

File: modules/genomic_browser/test/genomic_browserTest.php [hunk 1]
```diff
@@ -236,6 +225,7 @@ function testGenomicBrowserFilterEachTab()
     function _testPageUIs($url,$ui)
     {
         $this->safeGet($this->url . $url);
+        sleep(20);
```
Comment (kongtiaowang) [inline modules/genomic_browser/test/genomic_browserTest.php]: This sleep(20) need to be removed. Sorry, I forget to delete this line.

Comment (ridz1208) [reply modules/genomic_browser/test/genomic_browserTest.php]: @driusan @kongtiaowang done!

**Pull request merged**